RationalOptionPages.php

```diff
@@ -280,7 +280,7 @@ public function admin_init() {
 								$params['callback'] = array( $this, $params['callback'] );
 							}
 					
-							call_user_func_array( 'add_settings_field', $params );
+							add_settings_field( ...array_values( params ) );
```

Comment:
Sorry, I made a typo with my suggestion!

```suggestion
							add_settings_field( ...array_values( $params ) );
```

Reply:
Thank you for catching that! I did test these changes, but I just manually added them on my JN test site, rather than cloning the repo and pulling the branch :grimacing: